b2share/modules/access/ext.py

```diff
@@ -24,10 +24,6 @@
 """B2share access module extension."""
 
 from .loader import register_permissions_loader
-# FIXME: this is only needed because we don't use oauth2server's
-# server blueprint. Remove this once we integrate it.
-from invenio_oauth2server.views.server import login_oauth2_user
-
```

Comment:
Blueprint is not defined for B2SHARE v3 entrypoints, BUT login works fine without these lines.
Maybe invenio_oauth2server or b2share/oauthclient/b2access.py doesn't need these anymore.